.gitignore

```diff
@@ -7,3 +7,4 @@ build/
 UIP.egg-info/
 README
 .cache/
+uiplib/testPath.py
```

Comment:
Oh, I actually made it while I was working on a fix for permissions issue. Should I remove it?